### PR TITLE
[skip-revcheck] Generate ext/json class synopses from stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/json/jsonexception.xml [hunk 1]
```diff
@@ -28,34 +23,30 @@
...
-    <classsynopsisinfo role="comment">Methods</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.jsonexception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
--->
-<!-- If the exception extends another exception use this -->
-   <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
+    <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])">
+     <xi:fallback/>
+    </xi:include>
```
Comment (saundefined) [inline reference/json/jsonexception.xml:49]: `[skip-revcheck]` can not be applied

**Review by saundefined — COMMENTED** [review]
Thank you!

**Comment by kocsismate** [comment]
I accidentally deleted your comment.. sorry! Which is the diff that prevents skip-revcheck?

**Comment by saundefined** [comment]
`reference/json/jsonexception.xml` before:
```xml
<classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
  <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
</classsynopsis>
```
after:
```xml
<classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
  <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[not(@role='procedural')])">
    <xi:fallback/>
  </xi:include>
</classsynopsis>
```

I meant the added `<xi:fallback/>` 🤔

**Comment by kocsismate** [comment]
@cmb69 Can you please advise? ☝️:)

**Comment by cmb69** [comment]
Yes, probably better to drop the `skip-revcheck` in that case.

**Comment by kocsismate** [comment]
> Yes, probably better to drop the skip-revcheck in that case.

Thanks! Most probably some of my other PRs are also affected.

**Comment by cmb69** [comment]
Well, it's not that much of problem. If the translations can be build, the xi:fallback is not really necessary. Otherwise translators will notice soon enough. :)

**Pull request merged**